Documentation/ColumnsConfig/Type/Select/Tree/Index.rst

```diff
@@ -5,14 +5,35 @@
 selectTree
 ==========
 
-A field with the render type `selectTree` displays hierarchical data
-in a tree.
+
+The selectTree render type is used to display hierarchical data in a
```

Comment:
```suggestion
The :php:`selectTree` render type is used to display hierarchical data in a
```
selectTree is not an english word but a value that you use in TCA, therefore we mark it as PHP code